buildSrc/src/main/java/Config.kt

```diff
@@ -74,6 +74,8 @@ object Config {
         val servletApi = "javax.servlet:javax.servlet-api"
 
         val apacheHttpClient = "org.apache.httpcomponents.client5:httpclient5:5.0.3"
+        val apacheHttpCore = "org.apache.httpcomponents.core5:httpcore5:5.0.3"
+        val apacheHttpCoreH2 = "org.apache.httpcomponents.core5:httpcore5-h2:5.0.3"
```

Comment:
we could create an internal `val apacheHttpVersion = "5.0.3"` and reuse it

Reply:
The thing is that these versions are not really aligned. The same number is just an coincidence.